rma/views/rma_portal_templates.xml

```diff
@@ -132,12 +133,20 @@
                                         <span t-field="rma.picking_id"/>
                                     </div>
                                 </div>
-                                <div t-if="rma.product_id" class="row mb-2 mb-sm-1">
+                                <!-- We need to prevent access errors if the product is
+                                     unpublished. If not, we'll sudo to get the name-->
+                                <div t-if="rma.sudo().product_id" class="row mb-2 mb-sm-1">
                                     <div class="col-12 col-sm-4">
                                         <strong>Product</strong>
                                     </div>
                                     <div class="col-12 col-sm-8">
-                                        <span t-field="rma.product_id"/>
+                                        <!-- Don't depend on website -->
+                                        <t t-if="'website_published' in rma.sudo().product_id._fields and rma.sudo().product_id.website_published">
```

Comment:
Why not simply put the regular one? How does `sale` works with this?

Reply:
they put line.name, wich isn't valid here

Reply:
OK, but I think you should simply avoid the website_published stuff. Why `t-field` is not valid in every case?

Reply:
`website` adds this record rule: https://github.com/odoo/odoo/blob/12.0/addons/website/security/website_security.xml#L56-L61 So if the portal tries to access those records, it will raise an AcessError.

Another option to avoid this would be to create a compute field with the product name

Reply:
Well, you have put website.page record rule, which doesn't apply here, but I suppose you mean website_sale record rule over products. Let's put directly the t-esc over display name with the sudoed record

Reply:
Perfect. I let it like that